.github/workflows/publish_cpubuilder.yml

```diff
@@ -38,7 +38,8 @@ jobs:
         uses: docker/build-push-action@f2a1d5e99d037542a71f64918e516c093c6f3fc4
         with:
           context: .
-          file: dockerfiles/cpubuilder_ubuntu_jammy_x86_64.Dockerfile
+          file: dockerfiles/cpubuilder_ubuntu_jammy.Dockerfile
+          platforms: linux/amd64,linux/arm64
```

Comment:
https://github.com/iree-org/base-docker-images/actions/runs/10949036140/job/30401394373#step:5:135

```
 ERROR: Multi-platform build is not supported for the docker driver.
Switch to a different driver, or turn on the containerd image store, and try again.
Learn more at https://docs.docker.com/go/build-multi-platform/
Error: buildx failed with: Learn more at https://docs.docker.com/go/build-multi-platform/
```

Aww... I hit the same error locally while developing. The recommended steps in those docs worked for me, but I need to see what the docker/build-push-action needs.

Reply:
Fixed? https://github.com/iree-org/base-docker-images/commit/a61eb38120d704917e0b87855a9b106d224c91a8

So far so good: https://github.com/iree-org/base-docker-images/actions/runs/10949139831

Reply:
Yep, successful build and push. The workflow run time went up from 2 minutes to 20 minutes, in large part due to the ccache build from source.

Before: https://github.com/iree-org/base-docker-images/actions/runs/10948990985
After: https://github.com/iree-org/base-docker-images/actions/runs/10949139831

New package: https://github.com/iree-org/base-docker-images/pkgs/container/cpubuilder_ubuntu_jammy